train.py

```diff
@@ -61,10 +64,22 @@ def main():
   features.add_argparse_args(parser)
   args = parser.parse_args()
 
-  if not os.path.exists(args.train):
-    raise Exception('{0} does not exist'.format(args.train))
-  if not os.path.exists(args.val):
-    raise Exception('{0} does not exist'.format(args.val))
+  args.datasets = flatten_once(args.datasets)
+  if args.validation_datasets:
+    args.validation_datasets = flatten_once(args.validation_datasets)
```

Comment:
crashing with another error now: `TypeError: 'NoneType' object is not iterable`

but works when setting validation_datasets to be the same as training_datasets:

```python
if args.validation_datasets:
  args.validation_datasets = flatten_once(args.validation_datasets)
else:
  args.validation_datasets = args.datasets
```

Reply:
hmm, it looks like argparse sets it to None instead of an empty array, which is pretty dumb but ok. Should be fixed now, I hope...